scalalib/src/GenIdeaImpl.scala

```diff
@@ -499,15 +499,19 @@ case class GenIdeaImpl(evaluator: Evaluator,
     </module>
   }
   def libraryXmlTemplate(name: String, path: os.Path, sources: Option[os.Path], scalaCompilerClassPath: Loose.Agg[Path]): Elem = {
-    def url(path: os.Path): String = if (path.ext == "jar") "jar://" + path + "!/" else "file://" + path
+    def url(path: os.Path): String = {
+      if (path.ext == "jar") "jar://$PROJECT_DIR$/" + path.relativeTo(cwd) + "!/"
```

Comment:
We are using `cwd` which is relative to the Java process directory. Although I think it should work as expected in most cases, it would be good if we can use some better source for the current project directory.

Reply:
This `cwd` is passed in from elsewhere, and should be the base path for the `build.sc` file if I understand it right